scan/base.go

```diff
@@ -40,6 +42,350 @@ type base struct {
 	errs []error
 }
 
+type Command struct {
```

Comment:
__[Golint]__ exported type Command should have comment or be unexported

<sub>posted by Sider</sub>

Reply:
I used it in the sense of dependence. It made it easy to read